### PR TITLE
Anthony  remove second secure inputs

### PR DESCRIPTION
**Pull request by anthony-hull**
### Fixed Issues
$ https://github.com/Expensify/App/issues/6565
$ https://github.com/Expensify/App/issues/6823

### QA Steps
1. Create a new account
2. Click the link in the email
3. Verify the password form works (including some invalid password states)
4. Submit the form and verify it allows you to set the password with a valid password

1. Given being logged in
2. Navigate to the change password setting
3. Verify the password form works (including some invalid password states)
4. Submit the form and verify it allows you to change with a valid password

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [ ] Android

#### Web
image

#### Mobile Web
image

#### Desktop
image

#### iOS
image

#### Android
image

### CODE REVIEW

**Comment by parasharrajat** [comment]
I will wait for your tests..

File: src/pages/settings/PasswordPage.js [hunk 1]
```diff
@@ -49,12 +49,10 @@ class PasswordPage extends Component {
         this.state = {
             currentPassword: '',
             newPassword: '',
-            confirmNewPassword: '',
```
Comment (parasharrajat) [inline src/pages/settings/PasswordPage.js:52]: Not needed.

File: src/pages/settings/PasswordPage.js [hunk 2]
```diff
@@ -49,12 +49,10 @@ class PasswordPage extends Component {
...
             errors: {
                 currentPassword: false,
                 newPassword: false,
                 confirmNewPassword: false,
-                confirmPasswordMatch: false,
```
Comment (parasharrajat) [inline src/pages/settings/PasswordPage.js:57]: Not needed.

File: src/pages/settings/PasswordPage.js [hunk 3]
```diff
@@ -69,7 +67,6 @@ class PasswordPage extends Component {
             currentPassword: 'passwordPage.errors.currentPassword',
             confirmNewPassword: 'passwordPage.errors.confirmNewPassword',
             newPasswordSameAsOld: 'passwordPage.errors.newPasswordSameAsOld',
-            confirmPasswordMatch: 'setPasswordPage.passwordsDontMatch',
```
Comment (parasharrajat) [inline src/pages/settings/PasswordPage.js:72]: not needed.

**Review by parasharrajat — CHANGES_REQUESTED** [review]

File: src/pages/settings/PasswordPage.js (hunk 2 above)
Comment (parasharrajat) [reply src/pages/settings/PasswordPage.js:57]: hmm.,,,line above it.

File: src/pages/settings/PasswordPage.js (hunk 3 above)
Comment (parasharrajat) [reply src/pages/settings/PasswordPage.js:72]: line 68. Sorry

File: src/pages/settings/PasswordPage.js (hunk 2 above)
Comment (anthony-hull) [reply src/pages/settings/PasswordPage.js:57]: good catch, thanks

**Comment by parasharrajat** [comment]
@anthony-hull Could you please test it on Android? Thanks

**Comment by anthony-hull** [comment]
I've updated the post with the android screenshot

**Review by parasharrajat — APPROVED** [review]
LGTM. 

cc: @chiragsalian

**Review by chiragsalian — APPROVED** [review]
Changes LGTM.

**Comment by mountiny** [comment]
Edited the PR body to also link to the other issue this PR fixes.

**Pull request merged**